rosa_tutorials/rosa-mobb-using-aws-ack-tutorial.adoc

```diff
@@ -0,0 +1,162 @@
+:_content-type: ASSEMBLY
+[id=“rosa-mobb-using-aws-ack-tutorial”]
+= Tutorial: Using AWS Controllers for Kubernetes on ROSA
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-mobb-using-aws-ack-tutorial
+
+toc::[]
+
+//Mobb content metadata
+//Brought into ROSA product docs 2023-09-21
+//---
+//date: '2022-06-02'
+//title: Using AWS Controllers for Kubernetes (ACK) on ROSA
+//weight: 1
+//tags: ["AWS", "ROSA"]
+//authors:
+//  - Paul Czarkowski
+//  - Connor Wooley
+//---
+
+link:https://aws-controllers-k8s.github.io/community/docs/community/overview/[AWS Controllers for Kubernetes] (ACK) lets you define and use AWS service resources directly from Kubernetes. With ACK, you can take advantage of AWS-managed services for your Kubernetes applications without needing to define resources outside of the cluster or run services that provide supporting capabilities like databases or message queues within the cluster.
+
+ROSA clusters have a set of the ACK controllers in OperatorHub which makes it relatively easy to get started and use it. Caution should be taken as it is a tech preview product from AWS.
+
+This tutorial shows you how to use the ACK S3 controller as an example, but can be adapted for any other ACK controller that has an operator in the OperatorHub of your cluster.
+
+== Prerequisites
```

Comment:
`kubectl` is also a command in below steps, it should be prerequisite too.
And no step for cluster-admin creation. Let's also make it a prerequisite.